### PR TITLE
Boost: Add post purchase page

### LINKED ISSUE #23319: [Boost] Post-purchase landing page
Project: pc9hqz-iD-p2
Design: p1HpG7-f1L-p2 (please refer to Figma file for details)

Create a post-purchase landing page in wp-admin, as per the above Figma file - confirming the purchase and benefits with a Continue button leading back to the Boost dashboard.

### PR DESCRIPTION
**Pull request by eAdnan007**
Fixes #23319

#### Changes proposed in this Pull Request:

* This adds the landing page after boost was upgraded successfully.
* Improve the design of upgrade page.

#### Jetpack product discussion
pc9hqz-iD-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Go to 'Boost dashboard'
* Add `#/purchase-successful` to the URL
* Check the design on different screen sizes. dVyv3AO1fXG0gprXywMuMh-fi-2130%3A7342

image

image

### CODE REVIEW

**Review by karthikax — CHANGES_REQUESTED** [review]
Great work 🎉  A few suggestions:

- Margin's from all the `p` elements are gone. Changing to master fixes this. May need to look into why this is happening.
- container width in Figma is `1129px` and in diff is `1080px`. Does not seem like an important change though.
- There is a substantial spacing between `Dedicated email support` and `Continue` button in the design. This seems like a nice change to make.

**Comment by eAdnan007** [comment]
> * Margin's from all the `p` elements are gone. Changing to master fixes this. May need to look into why this is happening.

Can you try a hard refresh after building?

> container width in Figma is 1129px and in diff is 1080px. Does not seem like an important change though.

I was aiming it to match the other upgrade pages of related products.


> There is a substantial spacing between Dedicated email support and Continue button in the design. This seems like a nice change to make.

Improved

@karthikax

File: projects/plugins/boost/app/assets/src/css/main/common.scss [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
-  .my-#{$i} {
-	margin-top: $i * $spacing-unit;
-	margin-bottom: $i * $spacing-unit;
+	  .#{$acronym}#{$postfixes}-#{$i} {
+		@each $property-suffix in $properties {
+		  #{$property}-#{$property-suffix}: $spacing;
+		}
+	  }
+	}
   }
```
Comment (thingalon) [inline projects/plugins/boost/app/assets/src/css/main/common.scss:28]: 😲 I had no idea scss could do this sort of thing

**Review by thingalon — DISMISSED** [review]
👍 Great work! Looks just like the designs. The code is nice and clean too.

**Review by karthikax — APPROVED** [review]
> Can you try a hard refresh after building?
Looks like this also exists in `master` now. Unrelated to this PR.

Looks good to me to proceed. 👍

**Comment by eAdnan007** [comment]
r242957-wpcom

**Pull request merged**